### ISSUE TITLE
Apply in multi-loss setting

### ISSUE THREAD

**Issue by Wuziyi616** [JYChen18/RPMG#2]
Hi, thanks for open source this great work! I'm wondering how can apply this loss in a multi-loss training setting. Say in pose regression, we have a rotation loss and a translation loss. Can I just add the translation loss (e.g. MSE) and the RPMG loss together? So the only thing I will have to change is replacing the GT rotation with the GT computed from [this line](https://github.com/JYChen18/RPMG/blob/8d775beebeaa3988bd1ffd68cf7d6be107aedaa9/ModelNet_PC/code/train.py#L37). Is that correct? Have you tried RPMG in a multi-loss setup and are there any special things to care about e.g. the loss scaling? Thanks in advance!

**Comment by JYChen18** [comment]
Thanks for your attention to our work. You can find the experiment of camera relocalization in [this folder](https://github.com/JYChen18/RPMG/tree/main/poselstm-pytorch), which is very similar to your setting. The comment [here](https://github.com/JYChen18/RPMG/blob/main/utils/rpmg.py#L48) may also help you. As for the loss scaling, you can see the discussion in Section E.3 of our appendix.
By the way, the rotation computed from [this line](https://github.com/JYChen18/RPMG/blob/8d775beebeaa3988bd1ffd68cf7d6be107aedaa9/ModelNet_PC/code/train.py#L37) is a prediction of the network instead of a GT rotation.

**Comment by Wuziyi616** [comment]
Thanks for your prompt reply. That helps a lot!
